fluid-constructs/src/main/java/me/escoffier/fluid/constructs/ResponseCallback.java

```diff
@@ -0,0 +1,7 @@
+package me.escoffier.fluid.constructs;
+
+public interface ResponseCallback {
+
+    void respond(Object response);
```

Comment:
Should it return a Completable just in case the execution is async?

Reply:
Probably should be named `reply` (Agin, I'm not good with naming things - or everything would be called `Stuff`)

Reply:
I like the idea of returning completeable here! Otherwise we don't know when async reply has been completed. I will make the change.

Reply:
And +1 for `reply` :) I will change it as well.